ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main.yaml

```
@@ -97,9 +97,9 @@ tests:
       LCA_PULL_REF: pipeline:lifecycle-agent-operator
     env:
       OCP_BASE_IMAGE_SOURCE: ci
-      OCP_BASE_VERSION: "4.17"
+      OCP_BASE_VERSION: "4.19"
       OCP_TARGET_IMAGE_SOURCE: release
```

```suggestion
      OCP_TARGET_IMAGE_SOURCE: ci
```

I think we should use the `ci` releases and not the stable ones until `4.18` is GA. Otherwise [this](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/image-based/upgrade/seed/create/openshift-image-based-upgrade-seed-create-commands.sh#L27-L42) returns an empty release image URL.

```shell
$ curl -s "https://api.openshift.com/api/upgrades_info/graph?arch=amd64&channel=stable-4.18"                                                                           
{"version":1,"nodes":[],"edges":[],"conditionalEdges":[]}%
```

vs 

```shell
$ curl -s "https://amd64.ocp.releases.ci.openshift.org/graph?arch=amd64&channel=stable" | jq -r '.nodes[] | .version + " " + .payload' | sort -V | grep -F "4.18" | tail -n1
4.18.0-rc.2 registry.ci.openshift.org/ocp/release@sha256:23e31abd998aa80ee11344e6ad910d8283d4f2f973c79e9d7fecbd355be459ab
```